src/helpers/discord_utils.ts

```diff
@@ -376,13 +376,29 @@ async function sendMessageExceptionHandler(
  * @param messageContent - The MessageContent to send
  * @param file - The file to send
  * @param authorID - The author's ID
+ * @param interaction - The interaction
  */
 export async function sendMessage(
     textChannelID: string,
     messageContent: Eris.AdvancedMessageContent,
     file?: Eris.FileContent,
-    authorID?: string
+    authorID?: string,
+    interaction?: Eris.ComponentInteraction | Eris.CommandInteraction
 ): Promise<Eris.Message> {
+    if (interaction) {
+        if (!withinInteractionInterval(interaction)) {
+            return null;
+        }
+
+        try {
+            await interaction.createMessage(messageContent);
```

Comment:
should we return the message?

Reply:
it returns Promise<void> and we don't ever use the return value

Reply:
This function returns `Promise<Eris.Message>`

Reply:
https://abal.moe/Eris/docs/0.17.1/CommandInteraction#method-createMessage
![image](https://user-images.githubusercontent.com/13823855/180658614-4b9fb5fb-1545-4313-b453-ad5bb28750bf.png)
![image](https://user-images.githubusercontent.com/13823855/180658619-b09ca9d6-48b7-4109-b893-4481589846bd.png)

Reply:
Whoops, my bad